docs/resources/dsm_app.md

```diff
@@ -21,9 +66,51 @@ The following arguments are supported in the `dsm_app` resource block:
 
 * **name**: The Fortanix DSM App name
 * **default_group**: The Fortanix DSM group object id to be mapped to the app by default
-* _**other_group (optional)**_: The Fortanix DSM group object id the app needs to be assigned to
+* _**other_group (optional)**_: The Fortanix DSM group object id the app needs to be assigned to. If you want to 
+                                delete the existing groups from an app, remove the ids during update.
 * _**description (optional)**_: The description of the app 
 * _**new\_credential (optional)**_: Set this if you want to rotate/regenerate the API key. The values can be set as `True`/`False`
+* _**other_group_permissions(optional)**_: Incase if you want to change the default permissions of a new group.
+* _**mod_group_permissions (optional)**_: To modify the permissions of any existing group
+
+
+   other_group_permissions example:
+   
+   A varaiable should be declared. Here it is named as app_data. Please follow the below varaible reference 
+   to provide the permissions. For each group_id permissions should be given in a string format. Permissions
+   are separated by comma(",").
+
+   other_group_permissions = "${var.app_data.other_group}"
+   
+   variable "app_data" {
+    type = any
+    default = {
+        "other_group" = {
+            "<group_id>" : "SIGN,VERIFY,ENCRYPT,DECRYPT,WRAPKEY,UNWRAPKEY,DERIVEKEY,MACGENERATE,MACVERIFY,EXPORT,MANAGE,AGREEKEY,AUDIT,TRANSFORM"
+            "<group_id>" : "SIGN,ENCRYPT,DECRYPT,WRAPKEY,UNWRAPKEY,MACGENERATE,MACVERIFY,MANAGE,AGREEKEY,AUDIT,EXPORT"
+        }
+    }
+   }
+
+   mod_group_permissions example:
+
+   A varaiable should be declared. Here it is named as app_data. Please follow the below varaible reference 
```

Comment:
same comments about varaible and indentation

Reply:
Modified.